src/interactive/azext_interactive/azclishell/app.py

```diff
@@ -548,7 +549,7 @@ def handle_scenario(self, text):
             return
         self.scenario_repl(scenario)
 
-    def scenario_repl(self, scenario):
+    def scenario_repl(self, scenario, is_chatgpt=False):
```

Comment:
Please add some comments to explain the `is_chatgpt_render`
```suggestion
    def scenario_repl(self, scenario, is_chatgpt_render=False):
```